src/components/03-components/09-accordion/accordion.njk

```diff
@@ -1,142 +1,101 @@
     </button>
   </h2>
   <div id="a2" class="usa-accordion__content usa-prose">
-    <p>A well regulated Militia, being necessary to the security of a free State, the right of the people to keep and bear Arms, shall not be infringed.</p>
+    <p>{{body}}</p>
     {% render '@list' %}
-
   </div>
 
-  <!-- Use the accurate heading level to maintain the document outline -->
   <h2 class="usa-accordion__heading">
-    <button class="usa-accordion__button usa-accordion__block"
-      aria-expanded="false"
-      aria-controls="a3">
-      Third Amendment
+    <button class="usa-accordion__button" aria-expanded="false" aria-controls="a3">
+      {{header}}
     </button>
   </h2>
   <div id="a3" class="usa-accordion__content usa-prose">
-    <p>No Soldier shall, in time of peace be quartered in any house, without the consent of the Owner, nor in time of war, but in a manner to be prescribed by law.</p>
-
+    <p>{{body}}</p>
   </div>
 
-  <!-- Use the accurate heading level to maintain the document outline -->
   <h2 class="usa-accordion__heading">
-    <button class="usa-accordion__button usa-accordion__block"
-      aria-expanded="false"
-      aria-controls="a4">
-      Fourth Amendment
+    <button class="usa-accordion__button" aria-expanded="false" aria-controls="a4">
+      {{header}}
     </button>
   </h2>
   <div id="a4" class="usa-accordion__content usa-prose">
-    <p>The right of the people to be secure in their persons, houses, papers, and effects, against unreasonable searches and seizures, shall not be violated, and no Warrants shall issue, but upon probable cause, supported by Oath or affirmation, and particularly describing the place to be searched, and the persons or things to be seized.</p>
+    <p>{{body}}.</p>
 
   </div>
 
-  <!-- Use the accurate heading level to maintain the document outline -->
   <h2 class="usa-accordion__heading">
-    <button class="usa-accordion__button usa-accordion__block"
-      aria-expanded="false"
-      aria-controls="a5">
-      Fifth Amendment
+    <button class="usa-accordion__button" aria-expanded="false" aria-controls="a5">
+      {{header}}
     </button>
   </h2>
   <div id="a5" class="usa-accordion__content usa-prose">
-    <p>No person shall be held to answer for a capital, or otherwise infamous crime, unless on a presentment or indictment of a Grand Jury, except in cases arising in the land or naval forces, or in the Militia, when in actual service in time of War or public danger; nor shall any person be subject for the same offence to be twice put in jeopardy of life or limb; nor shall be compelled in any criminal case to be a witness against himself, nor be deprived of life, liberty, or property, without due process of law; nor shall private property be taken for public use, without just compensation.</p>
-
+    <p>{{header}}</p>
   </div>
-
 </div>
+#}
 
-<h6>Bordered</h6>
+<div class="usa-accordion usa-accordion--bordered" {% if multiselect %} aria-multiselectable="true"{% endif %}>
```

Comment:
add in components using accordions for consistency/to ensure proper configuration